### ISSUE TITLE
Why perform such operations on embeddings

### ISSUE THREAD

**Issue by 191220042** [spcl/QuaRot#67]
I see 

![Image](https://github.com/user-attachments/assets/a65fba53-c285-4413-a5e7-d4642db9ef95)
I wonder why perform such operations on embeddings and does such an operation affect the final output logits,thank you.

**Comment by sashkboos** [comment]
Thanks @191220042 For your issue.

This is because we want fuse the Hadamard into the output of the Embeddings to have it in the network.
